src/scenes/Preloader.ts

```diff
@@ -52,6 +52,9 @@ export class Preloader extends Scene {
             'truetype'
         );
 
+        // load background music
+        this.load.audio('background-music', 'assets/audio/background-music.ogg');
```

Comment:
https://caniuse.com/ogg-vorbis
looks like ie11 and safari 17 can't use ogg
i think we are good for this project
\o/